src/components/button/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import styled from "styled-components";
+import { colors } from "../../variables";
+const Buttons = ({
+  name,
+  className,
+  onClick,
+}: {
+  name: any;
+  className: any;
+  onClick?: () => void;
+}) => {
+  console.log(className);
+  return (
+    <Button className={className} onClick={onClick}>
+      {name}
+    </Button>
+  );
+};
+export default Buttons;
+const Button = styled.button`
```

Comment:
Move all the color codes, font-family to variable file. Hard coded string to .json file.

Reply:
Ya have Moved everything to variable file